modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py

```diff
@@ -251,6 +188,7 @@ def test_null_col(self, null_dtype):
             dtype={"a": "int64", "b": "int64", "c": null_dtype},
             skiprows=1,
         )
+        trigger_import(exp)
```

Comment:
Due to the following computations, we will have execution and therefore import triggered anyway for `exp`.

Reply:
moved import triggering to the place before casting to pandas and also added a check to do not import the same frame multiple times